projects/igniteui-angular/src/lib/grids/filtering/grid-filtering-row.component.ts

```diff
@@ -300,6 +299,8 @@ export class IgxGridFilteringRowComponent implements AfterViewInit {
      */
     public onInputClick() {
         if (this.column.dataType === DataType.Boolean) {
```

Comment:
Can we add an additional check if the dropdown is collapsed. It is not really related to the issue but dropdown opening is not working well when clicking on the input.

Reply:
@SAndreeva I added the check